src/pyoframe/io.py

```diff
@@ -134,7 +134,9 @@ def to_file(
         NamedMapper(Constraint) if use_var_names else Base62ConstMapper(Constraint)
     )
     for c in m.constraints:
-        const_map.add(c)
+        # Not adding empty constraints to mapping.
+        if len(c) > 0:
+            const_map.add(c)
```

Comment:
I think a better place to do this is directly within the `add()` function. For my understanding, when do we have constraints with no elements?

https://github.com/kjartan-at-bravos/pyoframe/blob/681ba4c64c63ee6135c27cf6016af0cbd05e725a/src/pyoframe/io_mappers.py#L37

Reply:
In my particular case, these are constraints that are empty when I create a model with a reduced number of hours. They would have definitions for other hours.

Reply:
I will move the check to .add()

Reply:
If you're able to add some simple tests for this use case that would be good. No worries if not!